src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisoners/BookingResourceIntTest.kt

```diff
@@ -174,4 +174,4 @@ class BookingResourceIntTest : IntegrationTestBase() {
   }
 }
 
-inline fun <reified B> WebTestClient.ResponseSpec.expectBodyResponse(): B = this.expectStatus().isOk.expectBody(B::class.java).returnResult().responseBody!!
+inline fun <reified B> WebTestClient.ResponseSpec.expectBodyResponse(): B = this.expectStatus().is2xxSuccessful.expectBody(B::class.java).returnResult().responseBody!!
```

Comment:
nice to be a lenient here. Test consumers expecting an 200 response won't case so long is it is a 2xx result - but you cam still independently test for say a 201 if you care (see below)